docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.9-slim AS builder
+
+RUN true \
+  && mkdir /app \
+  && apt update \
+  && apt install --no-install-recommends -y git \
+  && python -m venv /app/env \
+  && /app/env/bin/pip install wheel gunicorn
+
+ADD . /usr/local/src/bepasty
+RUN /app/env/bin/pip install /usr/local/src/bepasty
+
+# ---
+
+FROM python:3.9-slim
```

Comment:
3.11?

Reply:
updated but for now untested. should be able to have a look at the end of the week